tools/templates/legion-component/component.types.js

```diff
@@ -0,0 +1,14 @@
+module.exports = (componentName) => ({
+    content: `import { IStyle } from '@fluentui/react';
+
+    // eslint-disable-next-line @typescript-eslint/no-empty-interface
+    export interface I${componentName}Props {
+    }
+    
+    export interface I${componentName}Styles {
```

Comment:
do we gain anything from having the strict type definition on the styles? in some ways it's nice not to have to be explicit and simply be able to add properties in the style file. But if this lets us give it a type besides IStyle (like we do with sub component styles) then I'm for that.

Reply:
This definitely allows us to add other types (like IButtonStyles) but I haven't figured out how to use those with mergeStyleSets.